csharp/device/Microsoft.Azure.Devices.Client/DeviceAuthenticationWithSharedAccessPolicyKey.cs

```diff
@@ -3,23 +3,20 @@
 
 namespace Microsoft.Azure.Devices.Client
 {
+    using Microsoft.Azure.Devices.Client.Extensions;
     using System;
-    
```

Comment:
Something happen to all those comments... Obviously they belong there and should be kept.